features/polyExplorer/src/components/baseInfoPopUp/baseInfoPopUp.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useContext } from "react";
```

Comment:
Is there any chance this could be used somewhere else? And needed to be moved to poly-look?

Reply:
I'd say when we refactor it we move it to poly-look. This exact thing will be replaced in the next weeks so I'd say leave it here for now